com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/ShaderPass/VertMesh.hlsl

```diff
@@ -1,3 +1,6 @@
+#include "Packages/com.unity.render-pipelines.high-definition/Runtime/Debug/DebugDisplay.cs.hlsl"
```

Comment:
Why do you need those hearders here? it is not expected that we put  such header in VertMesh.hlsl